### PR TITLE
CHAD 13147: Matter Sensor Driver Profiles Cleanup

### PR DESCRIPTION
**Pull request by hcarter-775**
Many profiles in the drivers have outdated names that are unable to be mapped to. This change removes or renames outdated profile names to clean up this problem.

Jira ticket can be found [here](https://smartthings.atlassian.net/browse/CHAD-13417)

And this is the superset query that was used to make these decisions: https://superset.samsungiotanalytics.com/superset/sqllab?id=1155

### CODE REVIEW

File: drivers/SmartThings/matter-switch/profiles/matter-bridge.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-name: bridge
+name: matter-bridge
```
Comment (ctowns) [inline drivers/SmartThings/matter-switch/profiles/matter-bridge.yml:1]: I would think this profile is definitely being used in the field, so I don't think we can change the name safely

File: drivers/SmartThings/matter-sensor/profiles/matter-motion-battery-illuminance.yml [hunk 2]
```diff
@@ -1,16 +0,0 @@
-name: matter-motion-battery-illuminance
```
Comment (ctowns) [inline drivers/SmartThings/matter-sensor/profiles/matter-motion-battery-illuminance.yml:1]: Can we safely delete this? How did you confirm it is not being used? Same for the profile below

Comment (ctowns) [reply drivers/SmartThings/matter-sensor/profiles/matter-motion-battery-illuminance.yml:1]: If there is a superset query, you could link it here for posterity :)

**Comment by hcarter-775** [comment]
This is the superset query that was used to make these decisions: https://superset.samsungiotanalytics.com/superset/sqllab
cc: @ctowns

File: drivers/SmartThings/matter-sensor/profiles/matter-motion-battery-illuminance.yml (hunk 2 above)
Comment (hcarter-775) [reply drivers/SmartThings/matter-sensor/profiles/matter-motion-battery-illuminance.yml:1]: See superset query that is mentioned below

Comment (ctowns) [reply drivers/SmartThings/matter-sensor/profiles/matter-motion-battery-illuminance.yml:1]: That link just goes to the supserset website but not actually the query you used. You'll have to specifically share the query, which you should be able to find next to the `Run` button

Comment (hcarter-775) [reply drivers/SmartThings/matter-sensor/profiles/matter-motion-battery-illuminance.yml:1]: that should be updated now, sorry about that!

**Review by ctowns — APPROVED** [review]

**Pull request merged**